internal/integration_test/fuzz/wazerolib/nodiff.go

```diff
@@ -86,6 +86,10 @@ func requireNoDiff(wasmBin []byte, checkMemory bool, requireNoError func(err err
 	defer interpreter.Close(ctx)
 
 	compilerCompiled, err := compiler.CompileModule(ctx, wasmBin)
+	if err != nil && err.Error() == "importing empty named module is not allowed" {
```

Comment:
my other suggestion will break this as you'll need to use strings.HasSuffix "empty module name" ...